Hammerspoon/MJAppDelegate.m

```diff
@@ -129,6 +130,17 @@ - (BOOL)application:(NSApplication *)theApplication openFile:(NSString *)fileAnd
     return YES;
 }
 
+- (BOOL)application:(NSApplication *)application
+    continueUserActivity:(NSUserActivity *)userActivity
+      restorationHandler:(nonnull void (^)(NSArray<id<NSUserActivityRestoring>> *_Nullable))restorationHandler
+{
+  //NSLog(@"Open URL in NSUserActivityTypeBrowsingWeb");
+  if ([userActivity.activityType isEqualToString:NSUserActivityTypeBrowsingWeb]) {
+    [[NSWorkspace sharedWorkspace] openURL:userActivity.webpageURL];
+  }
+  return YES;
```

Comment:
Should this be unconditionally returning `YES`? The [docs](https://developer.apple.com/documentation/appkit/nsapplicationdelegate/1428471-application) suggest that we probably ought to return `YES` only inside the `if` statement, and otherwise return `NO`, right?

Reply:
@cmsj; if i were to correct this in a separate PR would that be ok? don't want to step on @mdbraber's toes, but also really want this in.